style.css

```diff
@@ -435,79 +427,96 @@ body {
 }
 
 .section-3 {
-	margin-bottom: 20px;
+	margin-bottom: 2%;
 }
 
 .wrapper-grid {
     display: grid;
-    grid-template-columns: repeat(7, 1fr);
+    grid-template-columns: 1fr 40% 1fr;
     grid-template-rows: repeat(2, 1fr);
-    grid-column-gap: 20px;
-    grid-row-gap: 20px;
+    grid-column-gap: 2%;
+    grid-row-gap: 3%;
 }
 
-.box1 { grid-area: 1 / 1 / 2 / 3; }
-.box2 { grid-area: 2 / 1 / 3 / 3; }
-.box3 { grid-area: 1 / 3 / 3 / 6; }
-.box4 { grid-area: 1 / 6 / 2 / 8; }
-.box5 { grid-area: 2 / 6 / 3 / 8; }
-
-.box3 {
-	position: relative;
-}
-
-.box-position {
-	position: absolute;
+.box1 { grid-area: 1 / 1 / 2 / 2; }
+.box2 { grid-area: 2 / 1 / 3 / 2; }
+.box3 { 
+    grid-area: 1 / 2 / 3 / 3;
+    display: flex;
+    position: relative;
 }
+.box4 { grid-area: 1 / 3 / 2 / 4; }
+.box5 { grid-area: 2 / 3 / 3 / 4; }
 
 .box-wrap {
     display: flex;
     flex-wrap: wrap;
-    justify-content: center;
-    align-items: center;
+    background-color: #EDEDED;
+    aspect-ratio: 1/1;
+    position: relative;
+}
+
+.img-wrap {
+    position: absolute;
+    width: 70%;
 }
 
 .box-text {
-	display: flex;
-    flex-wrap: wrap;
-    align-content: flex-end;
-    top: 87%;
-    left: 10%;
-    margin: 20px;
+	position: absolute;
```

Comment:
why do you use absolute value here? Isn't it possible to use relative value (without margins in percentage)?